Formula/ch341prog.rb

```diff
@@ -0,0 +1,17 @@
+class Ch341prog < Formula
+  desc "CLI tool for interfacing with ch341a USB SPI/IIC programmer"
+  homepage "https://github.com/setarcos/ch341prog"
+  url "https://github.com/setarcos/ch341prog.git", :using => :git, :tag => "v1.0b", :revision => "ac51c1ea52b9046ac634f052290b7036a49aff13"
```

Comment:
Please use the tarball since the git tree is not used here:
```rb
url "https://github.com/setarcos/ch341prog/archive/v1.0b.tar.gz"
sha256 "..."
```

Is v1.0b a stable release?